<launch>
    <arg name="tf_gripper_name" doc="name of the gripper joint in the tf tree"/>
    <arg name="tf_base_name" doc="name of the robot base in the tf tree"/>
    <arg name="chessboard_rows" doc="number of chess board rows"/>
    <arg name="chessboard_columns" doc="number of chessboard columns"/>
    <arg name="square_length" doc="chessboard square length (meter)"/>
    <arg name="marker_length" doc="Aruco marker length (meter)"/>
    
    <node name="hand_to_eye_calibrator" pkg="hand_eye" type="hand_to_eye.py" output="screen">
        <param name="rows" type="int" value="9"/>
        <param name="columns" type="int" value="6"/>
        <param name="square_length" type="double" value="0.02"/>
        <param name="marker_length" type="double" value="0.05"/>
        <param name="gripper" type="string" value="tool"/>
        <param name="robot_base" type="string" value="base_link"/>
    </node>
</launch>